.github/workflows/pr-check.yml

```
@@ -0,0 +1,33 @@
+      - name: Get dependencies
+        working-directory: ${{ env.WORKING_DIR }}
+        run: flutter pub get
+      
+      - name: Run Flutter Analyze
+        working-directory: ${{ env.WORKING_DIR }}
+        run: flutter analyze
+      
+      - name: Check Dart Format
+        working-directory: ${{ env.WORKING_DIR }}
+        run: |
+          dart format --set-exit-if-changed .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)